From 50c4194737fac6f3032d6287b77bd39e513367bf Mon Sep 17 00:00:00 2001
From: kanlihu <kanli.hu@intel.com>
Date: Tue, 7 Nov 2023 07:26:50 +0530
Subject: [PATCH] add edid bin to kernel firmware

add edid to kernel firmware
make DP-2 load 1080p edid

Tracked-On: OAM-112349
Signed-off-by: Kanli Hu <kanli.hu@intel.com>
---
 groups/graphics/auto/AndroidBoard.mk                            | 1 +
 .../gmin64/config-lts/linux-intel-lts2021/x86_64_defconfig      | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/groups/graphics/auto/AndroidBoard.mk b/groups/graphics/auto/AndroidBoard.mk
index 9354922..163d8d5 100644
--- a/groups/graphics/auto/AndroidBoard.mk
+++ b/groups/graphics/auto/AndroidBoard.mk
@@ -1,6 +1,7 @@
 {{#gen9+}}
 
 I915_FW_PATH := vendor/linux/firmware/i915
+I915_EDID_PATH := vendor/linux/firmware/edid
 
 # List of i915/guc_xxx.bin
 $(foreach t, $(patsubst $(I915_FW_PATH)/%,%,$(wildcard $(I915_FW_PATH)/kbl_guc_70*)) ,$(eval I915_FW += i915/$(t)) $(eval $(LOCAL_KERNEL) : $(PRODUCT_OUT)/vendor/firmware/i915/$(t)))
diff --git a/groups/kernel/gmin64/config-lts/linux-intel-lts2021/x86_64_defconfig b/groups/kernel/gmin64/config-lts/linux-intel-lts2021/x86_64_defconfig
index 7b58cc9..b0cca71 100644
--- a/groups/kernel/gmin64/config-lts/linux-intel-lts2021/x86_64_defconfig
+++ b/groups/kernel/gmin64/config-lts/linux-intel-lts2021/x86_64_defconfig
@@ -4158,7 +4158,7 @@ CONFIG_DRM_KMS_HELPER=y
 CONFIG_DRM_FBDEV_EMULATION=y
 CONFIG_DRM_FBDEV_OVERALLOC=100
 # CONFIG_DRM_FBDEV_LEAK_PHYS_SMEM is not set
-# CONFIG_DRM_LOAD_EDID_FIRMWARE is not set
+CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_DISPLAY_HELPER=y
 CONFIG_DRM_DISPLAY_DP_HELPER=y
 CONFIG_DRM_DISPLAY_HDCP_HELPER=y
-- 
2.40.0

